src/Symfony/Component/Yaml/Exception/UnderflowException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Yaml\Exception;
+
+/**
+ * Exception thrown when performing an invalid operation on an empty container, such as removing an element.
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class UnderflowException extends \UnderflowException implements ExceptionInterface
```

Comment:
Should it be `@internal` as it is always catched and replaced by a `ParseException`?